source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,37 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
```

Comment:
We should specify that ``updateOne()`` also takes an update document -- otherwise, it is unclear how you define the mutation that is applied.

Reply:
fixed